core/src/main/scala/toml/Codec.scala

```diff
@@ -9,12 +9,12 @@ trait Codec[A] {
     defaults: Codec.Defaults,
     index:    Int
   ): Either[Parse.Error, A]
+  private[toml] def optional: Boolean = false
```

Comment:
This field makes it easier to handle default value for missing field. toml does not have the concept of `null` or `nil`.